### PR TITLE
Noto Sans Nag Mundari: Version 1.000 added

### PR DESCRIPTION
**Pull request by simoncozens**
98b2fb2: [gftools-packager] Noto Sans Nag Mundari: Version 1.000 added

* Noto Sans Nag Mundari Version 1.000 taken from the upstream repo https://github.com/notofonts/nag-mundari.git at commit https://github.com/notofonts/nag-mundari/commit/.

### CODE REVIEW

**Comment by simoncozens** [comment]
Closes #6013.

File: ofl/notosansnagmundari/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  style: "normal"
+  weight: 400
+  filename: "NotoSansNagMundari[wght].ttf"
+  post_script_name: "NotoSansNagMundari-Regular"
+  full_name: "Noto Sans Nag Mundari Regular"
+  copyright: "\"Copyright 2023 The Noto Project Authors (https://github.com/notofonts/nag-mundari)"
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
```
Comment (RosaWagner) [inline ofl/notosansnagmundari/METADATA.pb:17]: @simoncozens missing subset, primary script, languages

File: ofl/notosansnagmundari/METADATA.pb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+category: "SANS_SERIF"
+date_added: "2023-03-16"
+fonts {
+  name: "Noto Sans Nag Mundari"
+  style: "normal"
+  weight: 400
+  filename: "NotoSansNagMundari[wght].ttf"
+  post_script_name: "NotoSansNagMundari-Regular"
+  full_name: "Noto Sans Nag Mundari Regular"
+  copyright: "\"Copyright 2023 The Noto Project Authors (https://github.com/notofonts/nag-mundari)"
```
Comment (RosaWagner) [inline ofl/notosansnagmundari/METADATA.pb:13]: duplicated \" in the copyright string

File: ofl/notosansnagmundari/METADATA.pb (hunk 1 above)
Comment (simoncozens) [reply ofl/notosansnagmundari/METADATA.pb:17]: Nag Mundari subset was not detected because we don't have one. I'll make one...

**Comment by thlinard** [comment]
HI @simoncozens 

Do you know why Noto fonts don't use the same `nameID` between the `STAT` table (`ValueNameID`) and the `fvar` table (`subfamilyNameID`)? To my knowledge, nothing that is specifically forbidden in the standard, but it causes issues in MS Office. And the non-Noto VFs here don't have that problem.

For exemple, for a non-Noto VF (Anybody[wdth,wght].ttf, recently merged), in TTX, one nameID:


```
> <STAT>
> […]
> <AxisValue index="12" Format="1">
>         <AxisIndex value="1"/>
>         <Flags value="0"/>
>         <ValueNameID value="262"/>  <!-- Medium -->
>         <Value value="500.0"/>
>       </AxisValue>
> […]
> </STAT>
> <fvar>
> […]
> <!-- Medium -->
>     <NamedInstance flags="0x0" subfamilyNameID="262">
>       <coord axis="wdth" value="100.0"/>
>       <coord axis="wght" value="500.0"/>
>     </NamedInstance>
```



For NotoSansNagMundari[wght].ttf, two different nameIDs:


```
> <STAT>
> […]
>       <AxisValue index="1" Format="1">
>         <AxisIndex value="0"/>
>         <Flags value="0"/>
>         <ValueNameID value="260"/>  <!-- Medium -->
>         <Value value="500.0"/>
>       </AxisValue>
> […]
> </STAT>
> <fvar>
> […]
> <!-- Medium -->
>     <NamedInstance flags="0x0" subfamilyNameID="263">
>       <coord axis="wght" value="500.0"/>
>     </NamedInstance>
```

**Comment by simoncozens** [comment]
> Do you know why Noto fonts don't use the same nameID between the STAT table (ValueNameID) and the fvar table (subfamilyNameID)? 

No, I don't know. I do a `gftools genstat` and a `gftools fix`, so this is nothing unusual compared to other fonts.

**Comment by thlinard** [comment]
> > Do you know why Noto fonts don't use the same nameID between the STAT table (ValueNameID) and the fvar table (subfamilyNameID)?
> 
> No, I don't know. I do a `gftools genstat` and a `gftools fix`, so this is nothing unusual compared to other fonts.

Odd. It's systematic. But thank you for your answer.

**Comment by RosaWagner** [comment]
made a pull subtree in #6059 and waiting for it to hit sandbox to merge this one

**Comment by simoncozens** [comment]
> waiting for it to hit sandbox to merge this one

We also need the subset changes (https://github.com/googlefonts/glyphsets/pull/105) to be merged by eng and synced their end before it's going to work.

**Comment by RosaWagner** [comment]
From the info in the different PRs I thought all was in sync, but this is still not showing a sample text in dev

**Comment by simoncozens** [comment]
It looks good to me.
Screenshot 2023-04-06 at 14 31 10

**Comment by RosaWagner** [comment]
@simoncozens 
So in sandbox the script is showing alright, except for the glyph page, the script is missing:

Capture d’écran 2023-04-20 à 11 32 46

**Comment by simoncozens** [comment]
I think that's a question for eng. The `nag-mundari` subset is defined in METADATA, but maybe they haven't actually done anything with it on the frontend.

**Pull request merged**